x-pack/plugins/cases/server/client/cases/validators.ts

```diff
@@ -59,8 +59,20 @@ export function validateCustomFieldTypesInRequest({
   }
 
   if (invalidCustomFieldKeys.length) {
+    const invalidCustomFieldLabels: string[] = [];
+
+    invalidCustomFieldKeys.forEach((invalidFieldKey) => {
+      const configuration = customFieldsConfiguration.find((item) => item.key === invalidFieldKey);
+
+      if (configuration !== undefined) {
+        invalidCustomFieldLabels.push(`"${configuration.label}"`);
```

Comment:
In the unlikely scenario where there is an invalid custom field but you cannot find the label the message would not contain anything. What about pushing the word `"Unknown"` to the labels array instead of not adding it at all?